src/Compilers/VisualBasic/Portable/Symbols/TypeSymbol.vb

```diff
@@ -740,5 +741,20 @@ Done:
         End Class
 
 #End Region
+
+        Private ReadOnly Property ITypeSymbol_NullableAnnotation As NullableAnnotation Implements ITypeSymbol.NullableAnnotation
+            Get
+                Return NullableAnnotation.None
+            End Get
+        End Property
+
+        Private Function ITypeSymbol_WithNullability(nullableAnnotation As NullableAnnotation) As ITypeSymbol Implements ITypeSymbol.WithNullableAnnotation
+            Return Me
```

Comment:
> We'll have to keep an eye out of this causes problems for the cross-language cases. ...

Sure, I think it would be better to have a real scenario rather than trying to prematurely complicate implementation.